darts/timeseries.py

```diff
@@ -1241,6 +1485,29 @@ def univariate_values(self, copy=True, sample=0) -> np.ndarray:
         else:
             return self._xa[:, 0, sample].values
 
+    def static_covariate_values(self, copy: bool = True) -> Optional[np.ndarray]:
```

Comment:
Shall this one maybe be called `static_covariates_values` (i.e., with an `s` at `covariates`)? For consistency.